2022-05 May.md

```diff
@@ -0,0 +1,3 @@
+# May 2022
+
+With `configd` out of the way I was finally able to focus on bumping the actual Apple open source releases we rely on to construct our own source-based SDK.  This went swimmingly. I've updated about half of the source releases and tracked down some of the headers we were missing, so our SDK will be more complete. So far I haven't encountered any major, new roadblocks—*knock on wood*.
```

Comment:
For reading purposes it's probably best to wrap this to let's say 100 chars per line, otherwise :tada:

Reply:
I agree but Discourse's markdown parsing is broken. It doesn't ignore newlines so a single paragraph needs to be on a single line, otherwise you get those weird line breaks that are common in emails.